reference/configuration/framework.rst

```diff
@@ -19,44 +19,6 @@ configured under the ``framework`` key in your application configuration.
     namespace and the related XSD schema is available at:
     ``https://symfony.com/schema/dic/symfony/symfony-1.0.xsd``
 
-annotations
```

Comment:
Was deprecated in 6.4, that's why there is no deprecated directive

https://github.com/symfony/symfony-docs/pull/18590

Enabling option on 7.0 was forbidden in  https://github.com/symfony/symfony/pull/51050